### DISCUSSION TITLE
Need help for setup my XTouch asap - Would pay for it

### DISCUSSION THREAD

**Discussion by DougCostanzo** [rsjaffe/MIDI2LR#567] [Problems]
Hi,

I was a happy MIDI2LR user, with my XTouch mini. I don't know why, it starts to not work properly. Knobs bumps adjustments or don't move. I leave a couple of threads here but sadly, without any answer. 

I have a week full of LR jobs to color correction and having it working again, would be great. If someone with a good knowledge in this Midi and the MIDI2 LR could have a quick online meeting with me, sharing screens and just put the things to work again, I could pay U$ 10,00 for this couple of minutes. 

Rsjaffe, sorry if I am doing something not allowed. It is not my intention. I am just desperate and knowing how great is the tool when working and not being able to find a solution by myself, is really frustrating. 

Anyone in the community that could help me?

Douglas Costanzo
contato@fdfotografia.com.br
whatsapp: 55 11 99951 9224

**Comment by jackal365** [comment]
Sounds like the hardware failure of the xtouch Mini itself?
Maybe check whether your xtouch Mini button is woek, or if possible, replace another XTouch mini and try it.

**Comment by DougCostanzo** [comment]
Hi, Jack. Thanks for you reply.

The mid is working. The problem now is, even though I installed several times the MIDI2LR, now is not show. As we can see on the image attached, it is enable but I don't have its screen or any signal as the plugin could be opened.

**Comment by DougCostanzo** [comment]
<img width="889" alt="Screen Shot 2021-12-07 at 17 53 57" src="https://user-images.githubusercontent.com/95177109/145104780-83144626-e07e-4e76-bcf0-7cb69f041646.png">

**Comment by DougCostanzo** [comment]
I don't if matter but I am using a macbook air M1 and LR Classic 11.0.1.
